plugins/inputs/gnmi/README.md

```diff
@@ -66,8 +69,9 @@ It has been optimized to support gNMI telemetry as produced by Cisco IOS XR (64-
     # heartbeat_interval = "60s"
 ```
 
-### Example Output
-```
+## Example Output
+
+```shell
```

Comment:
This is not a shell script or command
```suggestion
```text
```

Reply:
Sorry this was leftover from before I am guessing the readme was way before certain linting checks.

Reply:
No problem